devlib/module/sched.py

```diff
@@ -268,6 +268,58 @@ def set_kernel_attribute(self, attr, value, verify=True):
         path = '/proc/sys/kernel/sched_' + attr
         self.target.write_value(path, value, verify)
 
+    @property
+    @memoized
+    def has_debug(self):
+        if self.target.config.get('SCHED_DEBUG') != 'y':
+            logger.warn("target's kernel compiled with !CONFIG_SCHED_DEBUG")
```

Comment:
Why warn here? Should `SCHED_DEBUG` part of the condition for returning `True`?

Reply:
You can have 'file_exists' failing but just because the debugfs is not mounted... this was an attemp to warn the use in case debugfs is not available... but perhaps we should move it into a dedicated call and avoid explicit warnings?